nullaway/src/main/java/com/uber/nullaway/handlers/LibraryModelsHandler.java

```diff
@@ -1281,4 +1284,79 @@ private static Symbol.MethodSymbol lookupHandlingOverrides(
       return null;
     }
   }
+
+  /** Constructs Library Models from stubx files */
+  private static class ExternalStubxLibraryModels implements LibraryModels {
+
+    private static final Map<String, Map<String, Map<Integer, Set<String>>>> argAnnotCache;
+    private static final Map<String, Integer> upperBoundsCache;
```

Comment:
These fields should not be static!  That makes them mutable global state.  I think they can just be instance fields and everything will work.

Really, `StubxCacheUtil.getArgAnnotCache()` and `StubxCacheUtil.getUpperBoundCache()` should each return `ImmutableMap`s.  But given the size of this PR, we can look into that change as a follow up.